Default `HTTPSettings.enable_cleanup_closed` to `False`.
- CPython3.11 changes around SSLProto have made this quite unstable and prone to errors when dealing with unclosed TLS transports, which ends up in aiohttp calling close and abort twice.
